diff --git a/src/amr_criteria/refinement_package.cpp b/src/amr_criteria/refinement_package.cpp
index f2b56a989..015a26a16 100644
--- a/src/amr_criteria/refinement_package.cpp
+++ b/src/amr_criteria/refinement_package.cpp
@@ -101,107 +101,107 @@ void FirstDerivative(const AMRBounds &bnds, MeshData<Real> *md, const std::strin
                      const int &idx, ParArray1D<AmrTag> &amr_tags,
                      const Real refine_criteria_, const Real derefine_criteria_,
                      const int max_level_) {
-  const auto desc =
-      MakePackDescriptor(md->GetMeshPointer()->resolved_packages.get(), {field});
-  auto pack = desc.GetPack(md);
-  const int ndim = md->GetMeshPointer()->ndim;
-  const int nvars = pack.GetMaxNumberOfVars();
-
-  const Real refine_criteria = refine_criteria_;
-  const Real derefine_criteria = derefine_criteria_;
-  const int max_level = max_level_;
-  const int var = idx;
-  // get a scatterview for the tags that will use Kokkos::Max as the reduction operation
-  auto scatter_tags = amr_tags.ToScatterView<Kokkos::Experimental::ScatterMax>();
-  par_for_outer(
-      PARTHENON_AUTO_LABEL, 0, 0, 0, pack.GetNBlocks() - 1, bnds.ks, bnds.ke, bnds.js,
-      bnds.je,
-      KOKKOS_LAMBDA(team_mbr_t team_member, const int b, const int k, const int j) {
-        Real maxd = 0.;
-        par_reduce_inner(
-            inner_loop_pattern_ttr_tag, team_member, bnds.is, bnds.ie,
-            [&](const int i, Real &maxder) {
-              Real scale = std::abs(pack(b, var, k, j, i));
-              Real d = 0.5 *
-                       std::abs((pack(b, var, k, j, i + 1) - pack(b, var, k, j, i - 1))) /
-                       (scale + TINY_NUMBER);
-              maxder = (d > maxder ? d : maxder);
-              if (ndim > 1) {
-                d = 0.5 *
-                    std::abs((pack(b, var, k, j + 1, i) - pack(b, var, k, j - 1, i))) /
-                    (scale + TINY_NUMBER);
-                maxder = (d > maxder ? d : maxder);
-              }
-              if (ndim > 2) {
-                d = 0.5 *
-                    std::abs((pack(b, var, k + 1, j, i) - pack(b, var, k - 1, j, i))) /
-                    (scale + TINY_NUMBER);
-                maxder = (d > maxder ? d : maxder);
-              }
-            },
-            Kokkos::Max<Real>(maxd));
-        auto tags_access = scatter_tags.access();
-        auto flag = AmrTag::same;
-        if (maxd > refine_criteria && pack.GetLevel(b, 0, 0, 0) < max_level)
-          flag = AmrTag::refine;
-        if (maxd < derefine_criteria) flag = AmrTag::derefine;
-        // ScatterMax view will use an atomic_max to prevent race condition across k,j
-        // indices
-        tags_access(b).update(flag);
-      });
-  amr_tags.ContributeScatter(scatter_tags);
+  // const auto desc =
+  //     MakePackDescriptor(md->GetMeshPointer()->resolved_packages.get(), {field});
+  // auto pack = desc.GetPack(md);
+  // const int ndim = md->GetMeshPointer()->ndim;
+  // const int nvars = pack.GetMaxNumberOfVars();
+
+  // const Real refine_criteria = refine_criteria_;
+  // const Real derefine_criteria = derefine_criteria_;
+  // const int max_level = max_level_;
+  // const int var = idx;
+  // // get a scatterview for the tags that will use Kokkos::Max as the reduction operation
+  // auto scatter_tags = amr_tags.ToScatterView<Kokkos::Experimental::ScatterMax>();
+  // par_for_outer(
+  //     PARTHENON_AUTO_LABEL, 0, 0, 0, pack.GetNBlocks() - 1, bnds.ks, bnds.ke, bnds.js,
+  //     bnds.je,
+  //     KOKKOS_LAMBDA(team_mbr_t team_member, const int b, const int k, const int j) {
+  //       Real maxd = 0.;
+  //       par_reduce_inner(
+  //           inner_loop_pattern_ttr_tag, team_member, bnds.is, bnds.ie,
+  //           [&](const int i, Real &maxder) {
+  //             Real scale = std::abs(pack(b, var, k, j, i));
+  //             Real d = 0.5 *
+  //                      std::abs((pack(b, var, k, j, i + 1) - pack(b, var, k, j, i - 1))) /
+  //                      (scale + TINY_NUMBER);
+  //             maxder = (d > maxder ? d : maxder);
+  //             if (ndim > 1) {
+  //               d = 0.5 *
+  //                   std::abs((pack(b, var, k, j + 1, i) - pack(b, var, k, j - 1, i))) /
+  //                   (scale + TINY_NUMBER);
+  //               maxder = (d > maxder ? d : maxder);
+  //             }
+  //             if (ndim > 2) {
+  //               d = 0.5 *
+  //                   std::abs((pack(b, var, k + 1, j, i) - pack(b, var, k - 1, j, i))) /
+  //                   (scale + TINY_NUMBER);
+  //               maxder = (d > maxder ? d : maxder);
+  //             }
+  //           },
+  //           Kokkos::Max<Real>(maxd));
+  //       auto tags_access = scatter_tags.access();
+  //       auto flag = AmrTag::same;
+  //       if (maxd > refine_criteria && pack.GetLevel(b, 0, 0, 0) < max_level)
+  //         flag = AmrTag::refine;
+  //       if (maxd < derefine_criteria) flag = AmrTag::derefine;
+  //       // ScatterMax view will use an atomic_max to prevent race condition across k,j
+  //       // indices
+  //       tags_access(b).update(flag);
+  //     });
+  // amr_tags.ContributeScatter(scatter_tags);
 }
 
 void SecondDerivative(const AMRBounds &bnds, MeshData<Real> *md, const std::string &field,
                       const int &idx, ParArray1D<AmrTag> &amr_tags,
                       const Real refine_criteria_, const Real derefine_criteria_,
                       const int max_level_) {
-  const auto desc =
-      MakePackDescriptor(md->GetMeshPointer()->resolved_packages.get(), {field});
-  auto pack = desc.GetPack(md);
-  const int ndim = md->GetMeshPointer()->ndim;
-  const int nvars = pack.GetMaxNumberOfVars();
-
-  const Real refine_criteria = refine_criteria_;
-  const Real derefine_criteria = derefine_criteria_;
-  const int max_level = max_level_;
-  const int var = idx;
-  // get a scatterview for the tags that will use Kokkos::Max as the reduction operation
-  auto scatter_tags = amr_tags.ToScatterView<Kokkos::Experimental::ScatterMax>();
-  par_for_outer(
-      PARTHENON_AUTO_LABEL, 0, 0, 0, pack.GetNBlocks() - 1, bnds.ks, bnds.ke, bnds.js,
-      bnds.je,
-      KOKKOS_LAMBDA(team_mbr_t team_member, const int b, const int k, const int j) {
-        Real maxd = 0.;
-        par_reduce_inner(
-            inner_loop_pattern_ttr_tag, team_member, bnds.is, bnds.ie,
-            [&](const int i, Real &maxder) {
-              Real aqt = std::abs(pack(b, var, k, j, i)) + TINY_NUMBER;
-              Real qavg = 0.5 * (pack(b, var, k, j, i + 1) + pack(b, var, k, j, i - 1));
-              Real d = std::abs(qavg - pack(b, var, k, j, i)) / (std::abs(qavg) + aqt);
-              maxder = (d > maxder ? d : maxder);
-              if (ndim > 1) {
-                qavg = 0.5 * (pack(b, var, k, j + 1, i) + pack(b, var, k, j - 1, i));
-                d = std::abs(qavg - pack(b, var, k, j, i)) / (std::abs(qavg) + aqt);
-                maxder = (d > maxder ? d : maxder);
-              }
-              if (ndim > 2) {
-                qavg = 0.5 * (pack(b, var, k + 1, j, i) + pack(b, var, k - 1, j, i));
-                d = std::abs(qavg - pack(b, var, k, j, i)) / (std::abs(qavg) + aqt);
-                maxder = (d > maxder ? d : maxder);
-              }
-            },
-            Kokkos::Max<Real>(maxd));
-        auto tags_access = scatter_tags.access();
-        auto flag = AmrTag::same;
-        if (maxd > refine_criteria && pack.GetLevel(b, 0, 0, 0) < max_level)
-          flag = AmrTag::refine;
-        if (maxd < derefine_criteria) flag = AmrTag::derefine;
-        // ScatterMax view will use an atomic_max to prevent race condition across k,j
-        // indices
-        tags_access(b).update(flag);
-      });
-  amr_tags.ContributeScatter(scatter_tags);
+  // const auto desc =
+  //     MakePackDescriptor(md->GetMeshPointer()->resolved_packages.get(), {field});
+  // auto pack = desc.GetPack(md);
+  // const int ndim = md->GetMeshPointer()->ndim;
+  // const int nvars = pack.GetMaxNumberOfVars();
+
+  // const Real refine_criteria = refine_criteria_;
+  // const Real derefine_criteria = derefine_criteria_;
+  // const int max_level = max_level_;
+  // const int var = idx;
+  // // get a scatterview for the tags that will use Kokkos::Max as the reduction operation
+  // auto scatter_tags = amr_tags.ToScatterView<Kokkos::Experimental::ScatterMax>();
+  // par_for_outer(
+  //     PARTHENON_AUTO_LABEL, 0, 0, 0, pack.GetNBlocks() - 1, bnds.ks, bnds.ke, bnds.js,
+  //     bnds.je,
+  //     KOKKOS_LAMBDA(team_mbr_t team_member, const int b, const int k, const int j) {
+  //       Real maxd = 0.;
+  //       par_reduce_inner(
+  //           inner_loop_pattern_ttr_tag, team_member, bnds.is, bnds.ie,
+  //           [&](const int i, Real &maxder) {
+  //             Real aqt = std::abs(pack(b, var, k, j, i)) + TINY_NUMBER;
+  //             Real qavg = 0.5 * (pack(b, var, k, j, i + 1) + pack(b, var, k, j, i - 1));
+  //             Real d = std::abs(qavg - pack(b, var, k, j, i)) / (std::abs(qavg) + aqt);
+  //             maxder = (d > maxder ? d : maxder);
+  //             if (ndim > 1) {
+  //               qavg = 0.5 * (pack(b, var, k, j + 1, i) + pack(b, var, k, j - 1, i));
+  //               d = std::abs(qavg - pack(b, var, k, j, i)) / (std::abs(qavg) + aqt);
+  //               maxder = (d > maxder ? d : maxder);
+  //             }
+  //             if (ndim > 2) {
+  //               qavg = 0.5 * (pack(b, var, k + 1, j, i) + pack(b, var, k - 1, j, i));
+  //               d = std::abs(qavg - pack(b, var, k, j, i)) / (std::abs(qavg) + aqt);
+  //               maxder = (d > maxder ? d : maxder);
+  //             }
+  //           },
+  //           Kokkos::Max<Real>(maxd));
+  //       auto tags_access = scatter_tags.access();
+  //       auto flag = AmrTag::same;
+  //       if (maxd > refine_criteria && pack.GetLevel(b, 0, 0, 0) < max_level)
+  //         flag = AmrTag::refine;
+  //       if (maxd < derefine_criteria) flag = AmrTag::derefine;
+  //       // ScatterMax view will use an atomic_max to prevent race condition across k,j
+  //       // indices
+  //       tags_access(b).update(flag);
+  //     });
+  // amr_tags.ContributeScatter(scatter_tags);
 }
 
 void SetRefinement_(MeshBlockData<Real> *rc,
